### ISSUE TITLE
2.8.1 Patch Release Plan

### ISSUE THREAD

**Issue by joeparsons** [az-digital/az_quickstart#2923]
- #2897 
- #2880 
- #2894 
- #2910
- #2919
- #2895
- #2917 
- #2908

## Pick list
```
git cherry-pick a014676512579752238e3a0a95c3a9d69bd3dcc3
git cherry-pick a1c2231c209951cde631cfedf4946e47cd0295c4
git cherry-pick fd5b2572023d5510732e6596bc55019feec6575e
git cherry-pick 4004d069289cc7d31870ffbbe360bd25304db590
git cherry-pick 3cb69bd7be085b2075816bf57fe176b17d736fb2
git cherry-pick 08617a6609bcdf5fbd0d58b92f9b89f3b8ccd7c7
git cherry-pick 172e8692d60403a1e035f41586c438586af56604
git cherry-pick 6de0887adb399b9f9d11f6b70b070d2d59229c83
```

**Comment by joeparsons** [comment]
## Release notes snippet

`git log 2.8.0..HEAD --pretty='format:- %s'`
```
- Closes #2907 Make Manual Migration Lookup DB configurable. (#2908)
- Closes #2916 Exclude BigPipe module from Quickstart installs on Pantheon (#2917)
- Update drupal/viewsreference requirement from 2.0.0-beta6 to 2.0.0-beta7 (#2895)
- Closes #2918: Add AZ Bootstrap library dependency to node form overrides library. (#2919)
- Update drupal/entity_embed requirement from 1.4.0 to 1.5.0 (#2910)
- Update drupal/core-recommended requirement from 10.1.5 to 10.1.6 (#2894)
- Closes #2671: Enable copy options based on content type (#2880)
- Closes #2896 - Make white text visible while editing (#2897)
- Back to dev.
```

**Comment by joeparsons** [comment]
Release created: https://github.com/az-digital/az_quickstart/releases/tag/2.8.1
